_api_app/app/Http/routes.php

```diff
@@ -35,6 +36,8 @@
     $app->put('sites', 'SitesController@order');
     $app->delete('sites', 'SitesController@delete');
 
+    $app->get('multisite/render', 'SitesController@renderMultisteMenu');
```

Comment:
You can actually add a variable `{site}` to this route for easier testing. See `entries/render` route.